modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -299,3 +278,32 @@ mirror:
       shortestPath: true
       type: ocp
 ----
+
+[discrete]
+[id="oc-mirror-image-set-examples-mce-images-catalog-ocp_{context}"]
+== Use case: Including the multi-arch {product-title} images and catalog for {mce-short}
+
+The following `ImageSetConfiguration` file includes {mce} and all {product-title} versions starting at a minimum version of `4.17.0` to `4.17.1` in the channel.  
```

Comment:
Should this just say `at a minimum version of 4.17.0` because the to implies that this will need to be updated constantly. For example, 4.17.2 is shipping tomorrow.

Reply:
Or perhaps I'm not understanding something with the file and this should be re-written for clarity. "with a `minVersion` value of `4.17.0` and a `maxVersion` value of `4.17.1`"?

Reply:
@JoeAldinger here we are trying to highlight what the sample file content means but I think changing this to what you told `at a minimum version of 4.17.0` will make more sense.
cc @krishnanshruthi

Reply:
Updated